diagnostics/top_heaviness_metric/top_heaviness_ratio_calculation.py

```diff
@@ -0,0 +1,191 @@
+# 28 May top_heaviness_ratio_calculation.py
+#Python packages/ modules imported for the diagnostic
+# the sample monthly data is from ERA5 in July from 2000 to 2019 
+import os
+import xarray as xr
+import numpy as np
+from scipy import integrate
+import matplotlib.pyplot as plt
+from mpl_toolkits.basemap import Basemap
```

Comment:
Basemap is no longer supported.  Please rewrite the plotting using another package, such as Cartopy.

Reply:
We have updated our code with the Cartopy module.